templates/todo/projects/csharp-cosmos-sql/.repo/bicep/azure.yaml

```diff
@@ -3,12 +3,37 @@
 name: todo-csharp-cosmos-sql
 metadata:
   template: todo-csharp-cosmos-sql@0.0.1-beta
+workflows:
+  up: 
+    steps:
+      - azd: provision
+      - azd: deploy
 services:
   web:
     project: ../../web/react-fluent
-    dist: build
+    dist: dist
     language: js
     host: appservice
+    hooks:
+      # Creates a temporary `.env.local` file for the build command. Vite will automatically use it during build.
+      # The expected/required values are mapped to the infrastructure outputs.
+      # .env.local is ignored by git, so it will not be committed if, for any reason, if deployment fails.
+      # see: https://vitejs.dev/guide/env-and-mode
+      # Note: Notice that dotenv must be a project dependency for this to work. See package.json.
+      prepackage:
+        windows:
+          shell: pwsh
+          run: 'echo "VITE_API_BASE_URL=\"$env:API_BASE_URL\"" > .env.local ; echo "VITE_APPLICATIONINSIGHTS_CONNECTION_STRING=\"$env:APPLICATIONINSIGHTS_CONNECTION_STRING\"" >> .env.local'
```

Comment:
Is it possible that `.env.local` is a file that the user created and wanted to manage?

Reply:
it is possible, yeah.
The file is git-ignored by default for vite projects.
User might need to hanlde updating the hooks to keep the state for their app as they need. :)